src/Scanner.js

```diff
@@ -322,11 +331,7 @@ class DefaultScanner {
 
             // Only add the form if below the limit of forms per page
             if (this.forms.size < this.options.maxFormsPerPage) {
-                const f = new Form(parentForm, input, this.device, this.matching, this.shouldAutoprompt)
-                // Also only add the form if it hasn't self-destructed due to having too few fields
-                if (!f.isDestroyed) {
-                    this.forms.set(parentForm, f)
-                }
+                this.forms.set(parentForm, new Form(parentForm, input, this.device, this.matching, this.shouldAutoprompt))
```

Comment:
Always set the form instance here, because this is done within the `addInput` loop. The form may be deemed useless later, but for now we always create it to ensure future inputs detect it.

Reply:
This would still fill up the `forms` set (max 30). The side-effect of this being that if there's actual "useful" form later, it won't be detected. We probably want to increase the size of the `forms.set`?